runners/samza/src/main/java/org/apache/beam/runners/samza/translation/TranslationContext.java

```diff
@@ -283,4 +323,13 @@ public StoreIdGenerator getStoreIdGenerator() {
     sendFn.accept(new EndOfStreamMessage(null));
     return dummyInput;
   }
+
+  boolean doAttachMetricOp(Config config, boolean enableTransformMetric) {
```

Comment:
Is `shouldDoAttachMetricOp` a better naming?

I feel `doAttachMetricOp` has a naming collision with a method that does the actual action.